chapters/install/ide-speedup.md

```diff
@@ -0,0 +1,79 @@
+# Speed-up Jina App development in PyCharm and VSCode
+
+## Set up your developer environment for jina
+
+This chapter describes how to speed up your developer experience
+in Pycharm and VS Code IDE while using Jina.
+In particular we would focus on how to enable Intellisense for Jina YAML configs. Intellisense can provide code completion, showing number and arguments list, validating your YAML config.
+
+These features can help you build your Jina application much faster.
+Once you have the latest Jina version installed with PyPI, while creating a typical Jina application - two types of files need to be written: Python code and YAML config. The python
+file defines the entry point and customized logics while the YAML configs define the flow composition and the configuration of each individual executor.
+Depending on the cases and the depth of customization, you can develop the project in both ways. In Jina, it is even possible to build a completely code free project that only depends on the configs. Most IDEs have provided smart intelligence for python development - due to
+straightforward API type hints and 100% string coverage in jina framework, you can enjoy a pretty smooth development experience. However, for YAML configs, it doesn't help so much besides simple code highlight which could slow down your development speed as you have to navigate through the documentation of the code base in order to figure out which component
+or argument you really want to use.
+
+### Enable Intellisense for jina yaml configs in Pycharm:
```

Comment:
It's better to have the section title following the same pattern so that it is easier to follow.
```suggestion
### Enable Intellisense for jina yaml configs in Pycharm:
```

Reply:
I've applied `##` for all sub sections